app/models/user.rb

```diff
@@ -21,34 +21,39 @@ class User < ApplicationRecord
   has_many :badges, through: :badge_achievements
   has_many :collections, dependent: :destroy
   has_many :comments, dependent: :destroy
-  has_many :email_messages, class_name: "Ahoy::Message"
+  has_many :email_messages, class_name: "Ahoy::Message", dependent: :destroy
   has_many :github_repos, dependent: :destroy
   has_many :identities, dependent: :destroy
   has_many :mentions, dependent: :destroy
   has_many :messages, dependent: :destroy
-  has_many :notes, as: :noteable, inverse_of: :noteable
-  has_many :profile_pins, as: :profile, inverse_of: :profile
-  has_many :authored_notes, as: :author, inverse_of: :author, class_name: "Note"
+  has_many :notes, as: :noteable, inverse_of: :noteable, dependent: :delete_all
+  has_many :profile_pins, as: :profile, inverse_of: :profile, dependent: :delete_all
+  has_many :authored_notes, inverse_of: :author, class_name: "Note", foreign_key: :author_id, dependent: :delete_all # as: :author
   has_many :notifications, dependent: :destroy
   has_many :reactions, dependent: :destroy
   has_many :tweets, dependent: :destroy
   has_many :chat_channel_memberships, dependent: :destroy
   has_many :chat_channels, through: :chat_channel_memberships
   has_many :notification_subscriptions, dependent: :destroy
-  has_many :feedback_messages
-  has_many :rating_votes
+
+  has_many :offender_feedback_messages, class_name: "FeedbackMessage", inverse_of: :offender, foreign_key: :offender_id, dependent: :delete_all
```

Comment:
Specified multiple associations instead of one `feedback_messages` w/o options which failed because there is no `user_id` in the `feedback_messages` table.